x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -799,6 +799,29 @@ export const tasks: TelemetryTask[] = [
         })
       ).aggregations?.service_name.value;
 
+      const environmentsCount = (
+        await telemetryClient.search({
```

Comment:
https://github.com/elastic/kibana/blob/4b1b104738923bd475be6493533b68c5f770fa98/x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts#L779

Both index the same indices. it wouldn't be faster to run the query in one search?